code/modules/mob/living/living.dm

```diff
@@ -593,4 +593,7 @@ default behaviour is:
 	.+=ranged_accuracy_modifier
 
 /mob/living/is_organic()
-	return TRUE
\ No newline at end of file
+	return TRUE
+
+/mob/living/stasis_act()
+	set_extension(src, /datum/extension/stasis_effect/mob)
```

Comment:
this might need a little more complexity. If there's an existing status effect its duration should be refreshed